joplin/base/migrations/0033_auto_20180927_0500.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.0.8 on 2018-09-27 05:00
+
+from django.db import migrations
+from pprint import pprint
+from wagtail.core.blocks.stream_block import StreamValue
+
+def better_steps(apps, schema_editor):
+    '''
+    We can't import the models directly as they may be a newer
+    version than this migration expects. We use the historical versions.
+    '''
+    ServicePage = apps.get_model('base', 'ServicePage')
+    ServicePageStep = apps.get_model('base', 'ServicePageStep')
+
+    for page in ServicePage.objects.all():
+      pageSteps = []
+      for step in ServicePageStep.objects.all():
+        if step.page_id == page.id:
```

Comment:
I know there's a cleaner way to do this than iterating through everything and using an if statement, I was just tired of looking through the django docs by this point and it was working